traitsui/tests/test_layout.py

```diff
@@ -71,10 +80,8 @@ def test_qt_resizable_in_vgroup(self):
 
         from pyface import qt
 
-        with store_exceptions_on_all_threads():
-            dialog = VResizeDialog()
-            ui = dialog.edit_traits()
-
+        with store_exceptions_on_all_threads(), \
+                create_ui(VResizeDialog()) as ui:
             text = ui.control.findChild(qt.QtGui.QLineEdit)
```

Comment:
Since you are trying to make this more toolkit-independent, a better way to find the text control is one of either:
- `ui.ui_info.txt.control` (which works in simple cases like this); or
- `ui.get_editors('txt')[0].control`

That would make the only toolkit-dependent part actually getting the width from the widget.